### PR TITLE
create new attributes table with unique constraint

### PR DESCRIPTION
**Pull request by imabdulbasit**

### CODE REVIEW

File: crates/core/src/db/schema.rs [hunk 1]
```diff
@@ -8,7 +8,6 @@ table! {
         value -> Nullable<Text>,
         trait_type -> Nullable<Text>,
         id -> Uuid,
-        first_verified_creator -> Nullable<Varchar>,
```
Comment (ray-kast) [inline crates/core/src/db/schema.rs:11]: Did we mean to remove this?

Comment (kespinola) [inline crates/core/src/db/schema.rs:11]: Losing first_verified_creator? don't think we want to do that.

Comment (imabdulbasit) [reply crates/core/src/db/schema.rs]: I forgot to add it as it was a separate migration

**Review by ray-kast — APPROVED** [review]
We will have to keep an eye on dev to revert the migration being edited here.  @kespinola @dandlezzz

**Pull request merged**